circleguard/gui.pyw

```diff
@@ -61,15 +61,15 @@ class MainWindow(QWidget):
     def __init__(self):
         super(MainWindow, self).__init__()
 
-        self.tabWidget = QTabWidget()
+        self.tabwidget = QTabWidget()
```

Comment:
very bad, this stems from the fact that we're mixing camelCase and un_der_score style for naming variables - camelcase coming from qt (c++) and underscore coming from python obviously. There's no way we can get rid of the camelcase entirely because we're using qt but I think it would be good to have all variable names, even those for qt objects, follow underscore. This commit doesn't fix the problem - we're now following some obscure third style where everything is lowercase? It's better to stick to one of the two established styles lol.